js/windowMgr.js

```diff
@@ -741,6 +741,19 @@ function doCreateMainWindow(initialUrl, initialBounds, isCustomTitleBar) {
 
 }
 
+/**
+ * ELECTRON-956: App is not minimized upon "Configure Desktop Alert Position" modal when "Always on Top" = True
+ */
+app.on('browser-window-created', (event, window) => {
+    const parentWindow = window.getParentWindow();
+    if (parentWindow && !parentWindow.isDestroyed()) {
+        if (parentWindow.winName === 'main') {
```

Comment:
@KiranNiranjan will this work in case of email child pop-out windows when there is more than 1 level?

Reply:
@VishwasShashidhar This will work for more than 1 level. But this functionality will also disable `minimizable` and `maximizable` controls for the chat pop-outs

Reply:
Yes, if there is a parent associated with pop-out

Reply:
@KiranNiranjan @vrbsm If this disables `minimize` & `maximize` for our regular pop-outs, that would be a problem. We only need to apply this to all the special case windows like "About Symphony", "Notification Preferences" and so on.